The 11.3 linker does not support --version-script, so we suppress that here.

--- totem-pl-parser-3.26.5/plparse/meson.build~	2020-02-28 16:16:58.964869300 +0000
+++ totem-pl-parser-3.26.5/plparse/meson.build	2021-03-12 18:50:41.549636460 +0000
@@ -69,7 +69,6 @@
                        include_directories: [config_inc, totemlib_inc],
                        c_args: plparser_cflags,
                        dependencies: totem_pl_parser_deps,
-                       link_args : symbol_link_args,
                        link_with: totem_glibc_lib,
                        link_depends : symbol_map,
                        version: plparse_libversion,
@@ -104,7 +103,6 @@
                             include_directories: [config_inc, totemlib_inc],
                             c_args: plparser_cflags + ['-DTOTEM_PL_PARSER_MINI'],
                             dependencies: gio_dep,
-                            link_args : mini_symbol_link_args,
                             link_with: totem_glibc_lib,
                             link_depends : mini_symbol_map,
                             version: plparse_libversion,
